src/ui/Backdrop/Backdrop.ts

```diff
@@ -0,0 +1,74 @@
+import {Component} from '../Base/Component';
+import {ComponentOptions} from '../Base/ComponentOptions';
+import {IComponentBindings} from '../Base/ComponentBindings';
+import {Initialization, IInitializationParameters} from '../Base/Initialization';
+import {IResultsComponentBindings} from '../Base/ResultsComponentBindings';
+import {IQueryResult} from '../../rest/QueryResult';
+
+export interface IBackdropOptions {
+  imageField: string;
+  overlayColor?: string;
+  overlayGradient?: boolean;
+}
+
+/**
+ * This component is used to render an image URL (contained in a result's field)
+ * as a background image.
+ */
+export class Backdrop extends Component {
+  static ID = 'Backdrop';
+
+  /**
+   * @componentOptions
+   */
+  static options: IBackdropOptions = {
+    /**
+     * Specifies the field from which the background image will be pulled.
+     */
+    imageField: ComponentOptions.buildStringOption({ required: true }),
+    /**
+     * If specified, this color will be overlaid on top of the background image.
+     * It needs to be declared as a CSS color (be sure to use RGBA with an alpha
+     * value lower than 1 in order to be able to see the image behind).
+     *
+     * Example value : "`rgba(101, 123, 76, 0.5)`"
+     */
+    overlayColor: ComponentOptions.buildStringOption(),
```

Comment:
Hum, I'm thinking we should probably have a dedicated "color" type option in Component options, which would ensure we are consistent for every component.

And would also help the interface editor (by proposing a color picker for those type of option by default)